tools/code_timer.py

```diff
@@ -0,0 +1,34 @@
+# code_timer.py
```

Comment:
Needs a SPDX-license-identifier comment at the top.

Reply:
fixed in [413b4e3](https://github.com/daniel-thompson/wasp-os/pull/309/commits/413b4e36aab2c27babcd8896d4e4946a78e3cc75)